applications/HDF5Application/custom_io/hdf5_properties_io.cpp

```diff
@@ -12,60 +32,78 @@ namespace HDF5
 namespace Internals
 {
 
-void ReadProperties(File& rFile, std::string const& rPrefix, PropertiesContainerType& rProperties)
+void ReadProperties(
+    File& rFile,
+    const std::string& rPrefix,
+    PropertiesContainerType& rProperties)
 {
     KRATOS_TRY;
 
-    Vector<int> prop_ids;
-    if (rFile.HasAttribute(rPrefix + "/Properties", "Ids"))
-    {
-        rFile.ReadAttribute(rPrefix + "/Properties", "Ids", prop_ids);
-    }
-    for (auto pid : prop_ids)
-    {
-        std::stringstream pstream;
-        pstream << rPrefix << "/Properties/(" << pid << ")";
-        std::string path = pstream.str();
-
-        PropertiesType::ContainerType& r_data = rProperties[pid].Data();
-        Internals::ReadDataValueContainer(rFile, path, r_data);
-    }
+    KRATOS_ERROR_IF_NOT(rProperties.empty()) << "The properties container is not empty.";
 
-    KRATOS_CATCH("");
-}
+    IndexType start_index, block_size;
+    std::tie(start_index, block_size) = StartIndexAndBlockSize(rFile, rPrefix + "/Properties/");
```

Comment:
Structured bindings ftw!


```suggestion
    const auto [start_index, block_size] = StartIndexAndBlockSize(rFile, rPrefix + "/Properties/");
```

Reply:
not addressed